CHANGELOG.rst

```diff
@@ -17,7 +17,11 @@ Unreleased
 ----------
 * nothing unreleased
 
-[4.18.4]
+[4.19.5]
+--------
+* feat: remove unencrypted user credentials from view and test
```

Comment:
We are not removing credentials. Since we are replacing the original column references to encrypted column, I think "replace" would be a better terminology to use here.